addons/l10n_ar_withholding/wizards/account_payment_register.py

```diff
@@ -0,0 +1,199 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import models, fields, api, Command, _
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountPaymentRegisterWithholding(models.TransientModel):
+    _name = 'account.payment.register.withholding'
+    _description = 'Payment register withholding lines'
+
+    payment_register_id = fields.Many2one('account.payment.register', required=True, ondelete='cascade')
+    currency_id = fields.Many2one(related='payment_register_id.currency_id')
+    name = fields.Char(string='Number', required=False, default='/')
+    tax_id = fields.Many2one('account.tax', required=True)
+    withholding_sequence_id = fields.Many2one(related='tax_id.l10n_ar_withholding_sequence_id')
+    base_amount = fields.Monetary(required=True, compute='_compute_base_amount', store=True, readonly=False)
+    amount = fields.Monetary(required=True, compute='_compute_amount', store=True, readonly=False)
```

Comment:
why is there no company_id field? (there is one in the payment wizard as well)

Reply:
done